proto/cosmos/staking/v1beta1/tx.proto

```diff
@@ -45,6 +45,10 @@ service Msg {
   // parameters.
   // Since: cosmos-sdk 0.47
   rpc UpdateParams(MsgUpdateParams) returns (MsgUpdateParamsResponse);
+
+  // SendEventRecord defines an operation for updating the x/staking module
+  // parameters.
+  rpc SendEventRecord(MsgEventRecord) returns (MsgSendEventRecordResponse);
```

Comment:
How are you testing? Broadcasting the same event to all nodes? Otherwise behavior of extend vote may be non-deterministic.

Reply:
As discussed yesterday, a validator would  be leveraging the CLI to broadcast the tx like so : 
`build/simd tx staking event  --from cosmos1czfy9xfjzgphjjufmhfcxx2j7450qkmfuavs65 --keyring-backend=test --keyring-dir=.testnets/chain-YcCES5/node0/simcli --chain-id chain-YcCES5`

Reply:
Is this the only case @Raneet10 (leveraging the CLI I mean)?